front/src/config/i18n/en.json

```diff
@@ -791,6 +791,18 @@
         "saveLabel": "Save configuration"
       }
     },
+    "oauth2": {
+      "buttonConnect": "Connect now",
+      "apiKeyLabel": "Then, enter your client_id and secret_id key here:",
+      "errorEmptyConfig": "Client Id and Secret Id must not be empty !",
+      "errorServiceId": "Service Id of current integration must not be found !",
```

Comment:
Are you sure service id can be 'not found'? As an integration is linked to a service.
For more, in english, no space before `!`.

Reply:
I remove space.
I keep errorServiceId, is an error message can't be appear except in case of bad configuration of developper